dns_test.go

```diff
@@ -204,6 +205,51 @@ func TestListDNSRecords(t *testing.T) {
 	}
 }
 
+func TestUpdateDNSRecord(t *testing.T) {
+	client, server, _ := NewClientForTesting(map[string]string{
+		"/v2/dns/edc5dacf-a2ad-4757-41ee-c12f06259c70/records/76cc107f-fbef-4e2b-b97f-f5d34f4075d3": `{
+		  "id": "76cc107f-fbef-4e2b-b97f-f5d34f4075d3",
+		  "created_at": "2019-04-11T12:47:56.000+01:00",
+		  "updated_at": "2019-04-11T12:47:56.000+01:00",
+		  "account_id": null,
+		  "domain_id": "edc5dacf-a2ad-4757-41ee-c12f06259c70",
+		  "name": "email",
+		  "value": "10.0.0.1",
+		  "type": "mx",
+		  "priority": 10,
+		  "ttl": 600
+		}`,
+	})
+	defer server.Close()
+	rc := &DNSRecordConfig{DNSDomainID: "edc5dacf-a2ad-4757-41ee-c12f06259c70", Name: "email"}
+	d := &DNSDomain{ID: "edc5dacf-a2ad-4757-41ee-c12f06259c70", AccountID: "1", Name: "example.com"}
```

Comment:
With the change to the method signature, this object shouldn't be needed, but you will need to put `DNSDomainID` in the DNS Record object here.